ansible/run_galaxy_config.sh

```diff
@@ -3,3 +3,38 @@ sudo apt-get update -y && sudo apt-get install -y --no-install-recommends ansibl
 ansible-playbook -i "localhost," -c local ansible/set-galaxy-config-values.yaml
 echo "Playbook for galaxy config run."
 
+# if admin email, api and password env variables are set, then start galaxy, run user creation and stop galaxy
+if [ ! -z $GALAXY_ADMIN_EMAIL ] && [ ! -z $GALAXY_ADMIN_PASSWORD ] && [ ! -z $GALAXY_API_KEY ]; then
+	# ini file will be set already at this point with these variables, we only need to start galaxy
+	# wait for it to be available for API calls and create the new user
+	echo "Set up galaxy database for admin user setup..."
+	./run.sh --daemon
```

Comment:
It would be defensive to `rm -f paster.pid` before calling `run.sh` to avoid looping forever if the file ever doesn't get written properly (not sure whether this error case is currently possible).